test/parallel/test-assert.js

```diff
@@ -55,6 +55,52 @@ assert.throws(() => a.ok(false), a.AssertionError, 'ok(false)');
   assert.ok(threw, 'Error: ok(false)');
 }
 
+// Throw message if the message is instanceof Error.
```

Comment:
It is actually not about instanceof error in this case but more about being a native error?
```suggestion
// Thrown error should be the passed through error instance of the native error
// class, created in a different realm.
```